DigitalOffice/Services/TimeManagementService/Database/Entities/DbLeaveTypes.cs

```diff
@@ -1,8 +1,7 @@
-﻿using System;
 namespace LT.DigitalOffice.TimeManagementService.Database.Entities
 {
-   public enum LeaveType
-   {
+    public enum LeaveType
+    {
         Vacation = 0,
         SickLeave = 1,
         Training = 2,
```

Comment:
The name of the file should be without "S": DbLeaveType, isn't it?

Reply:
Yes. This is needed in another pull requestn other pull request